support-frontend/assets/helpers/errorReasons.js

```diff
@@ -27,15 +30,18 @@ function appropriateErrorMessage(errorReason: ?ErrorReason): ?string {
     case 'insufficient_funds':
       return 'The transaction was declined due to insufficient funds in your account. Please use a different card or contact your bank.';
     case 'payment_details_incorrect':
-      return 'An error occurred while trying to process your payment. Please double check your card details and try again. Alternatively, try another card or payment method.';
```

Comment:
this has changed the error messages for stripe as well, do we want to do this?

Reply:
good spot... seem to have accidentally blatted that message completely :/